Azure/sonic-platform-daemons
# [xcvrd] Don't log unnecessary messages upon empty transceiver change event

Eliminates unnecessary logs of the form:
```
xcvrd: Got event True {} in state 1
xcvrd: mapping from True {'-1': 'system_become_ready'} to system_become_ready
xcvrd: Got system_become_ready in normal state, ignored
```

What I did:
    Don't need to do anything when the port_dict of transceiver_change is empty

How to verify:
    Run on DUT to make sure xcvrd can detect tranceiver eeprom correctly and
    there is no unmeaning logs anymore.

Signed-off-by: chiourung_huang <chiourung_huang@edge-core.com>
